### ISSUE TITLE
Training suddenly stopped

### ISSUE THREAD

**Issue by hely4609** [open-mmlab/mmdetection3d#1191]
Env

sys.platform: linux
Python: 3.7.11 (default, Jul 27 2021, 14:32:16) [GCC 7.5.0]
CUDA available: True
GPU 0,1: GeForce RTX 2070 SUPER
CUDA_HOME: /usr/local/cuda-10.1
NVCC: Cuda compilation tools, release 10.1, V10.1.243
GCC: gcc (Ubuntu 7.5.0-3ubuntu1~18.04) 7.5.0
PyTorch: 1.6.0
PyTorch compiling details: PyTorch built with:
  - GCC 7.3
  - C++ Version: 201402
  - Intel(R) oneAPI Math Kernel Library Version 2021.4-Product Build 20210904 for Intel(R) 64 architecture applications
  - Intel(R) MKL-DNN v1.5.0 (Git Hash e2ac1fac44c5078ca927cb9b90e1b3066a0b2ed0)
  - OpenMP 201511 (a.k.a. OpenMP 4.5)
  - NNPACK is enabled
  - CPU capability usage: AVX2
  - CUDA Runtime 10.1
  - NVCC architecture flags: -gencode;arch=compute_37,code=sm_37;-gencode;arch=compute_50,code=sm_50;-gencode;arch=compute_60,code=sm_60;-gencode;arch=compute_61,code=sm_61;-gencode;arch=compute_70,code=sm_70;-gencode;arch=compute_75,code=sm_75;-gencode;arch=compute_37,code=compute_37
  - CuDNN 7.6.3
  - Magma 2.5.2
  - Build settings: BLAS=MKL, BUILD_TYPE=Release, CXX_FLAGS= -Wno-deprecated -fvisibility-inlines-hidden -DUSE_PTHREADPOOL -fopenmp -DNDEBUG -DUSE_FBGEMM -DUSE_QNNPACK -DUSE_PYTORCH_QNNPACK -DUSE_XNNPACK -DUSE_VULKAN_WRAPPER -O2 -fPIC -Wno-narrowing -Wall -Wextra -Werror=return-type -Wno-missing-field-initializers -Wno-type-limits -Wno-array-bounds -Wno-unknown-pragmas -Wno-sign-compare -Wno-unused-parameter -Wno-unused-variable -Wno-unused-function -Wno-unused-result -Wno-unused-local-typedefs -Wno-strict-overflow -Wno-strict-aliasing -Wno-error=deprecated-declarations -Wno-stringop-overflow -Wno-error=pedantic -Wno-error=redundant-decls -Wno-error=old-style-cast -fdiagnostics-color=always -faligned-new -Wno-unused-but-set-variable -Wno-maybe-uninitialized -fno-math-errno -fno-trapping-math -Werror=format -Wno-stringop-overflow, PERF_WITH_AVX=1, PERF_WITH_AVX2=1, PERF_WITH_AVX512=1, USE_CUDA=ON, USE_EXCEPTION_PTR=1, USE_GFLAGS=OFF, USE_GLOG=OFF, USE_MKL=ON, USE_MKLDNN=ON, USE_MPI=OFF, USE_NCCL=ON, USE_NNPACK=ON, USE_OPENMP=ON, USE_STATIC_DISPATCH=OFF, 

TorchVision: 0.7.0
OpenCV: 4.5.5
MMCV: 1.3.8
MMCV Compiler: GCC 7.3
MMCV CUDA Compiler: 10.1
MMDetection: 2.14.0
MMSegmentation: 0.14.1
MMDetection3D: 0.17.0+a582cbd

------------
![Screenshot from 2022-01-24 09-23-54](https://user-images.githubusercontent.com/48938847/150726477-d9c860e5-e27a-465e-ba88-e5224e7b41fc.png)

![Screenshot from 2022-01-24 09-20-09](https://user-images.githubusercontent.com/48938847/150726484-0ae3c408-8447-4ef3-9990-c2e3a6378943.png)

I don't know why the training stopped.
It's okay at first, but after a while, there was a sudden error.

**Comment by ZCMax** [comment]
You can refer to this issue: https://github.com/open-mmlab/mmdetection3d/issues/306

**Comment by hely4609** [comment]
@ZCMax Thanks for reply. 
I tried to improve according to your advice, but there was a new error
![Screenshot from 2022-01-26 11-32-11](https://user-images.githubusercontent.com/48938847/151094736-234088d8-0534-495c-bd97-e16b7244cb74.png)
![Screenshot from 2022-01-26 11-31-53](https://user-images.githubusercontent.com/48938847/151094772-49bb47b8-a339-4bcf-b68e-a8c010a1b962.png)

I'm sorry to keep bothering you about errors, but I'm eager for this to be resolved

**Comment by Tai-Wang** [comment]
It might be caused by your nan loss. Please first check it.

**Comment by Tai-Wang** [comment]
Close due to inactivity.
